sample/src/main/java/com/prolificinteractive/materialcalendarview/sample/DynamicSettersActivity.java

```diff
@@ -24,6 +27,7 @@
 
     @InjectView(R.id.textView) TextView textView;
     @InjectView(R.id.calendarView) MaterialCalendarView widget;
+    private int currentTileSize = 44;
```

Comment:
does the 44 represent the default value? Declaring a constant `DEFAULT_SIZE` and then initializing the `currentTileSize` with that may be more readable.